lib/upgrade.js

```diff
@@ -21,9 +24,20 @@ async function upgrade() {
 
     await jsonUtils.writeJson('package.json', packageJson);
 
-    await cleanup.deleteDependencies();
+    if (runInstall) {
+      logger.info('Done updating package.json.\n');
+
+      const settings = {};
+      if (logger.logLevel === 'verbose') {
+        settings.stdio = 'inherit';
+      }
+
+      await npmInstall(settings);
```

Comment:
Run `npm install` instead of deleting `node_modules` and `package-lock.json`. This makes the upgrade experience faster and less "destructive" to the normal NPM workflow.